entity-types/infra-gcpaiplatformpipelinejob/golden_metrics.yml

```diff
@@ -0,0 +1,18 @@
+JobDuration:
+  title: Runtime seconds of the pipeline job being executed (from creation to end)
+  unit: SECONDS
+  queries:
+    gcpSample:
+      select: average(`pipelinejob.Duration`)
+      from: GcpVertexAiPipelineJobSample
+      eventId: entityGuid
+      eventName: entityName
+TaskCompletedCount:
```

Comment:
Could you start the metricNames with lowercase [like in the documentation](https://github.com/newrelic/entity-definitions/blob/main/docs/entities/golden_metrics.md). It should be `jobDuration`